### PR TITLE
Add Eternal moments and shards

### PR DESCRIPTION
**Pull request by hexcowboy**

### CODE REVIEW

**Review by niveda-krish — APPROVED** [review]

File: src/cadence/scripts/getNFTs.cdc [hunk 1]
```diff
@@ -401,16 +403,53 @@ pub fun getEverbloom(owner: PublicAccount, id: UInt64): NFTData? {
     )
 }
 
+// https://flow-view-source.com/mainnet/account/0xc38aea683c0c4d38/contract/Eternal
```
Comment (niveda-krish) [inline src/cadence/scripts/getNFTs.cdc:406]: Could you share a link to the testnet as well?

Comment (hexcowboy) [reply src/cadence/scripts/getNFTs.cdc:406]: https://github.com/alchemyplatform/alchemy-flow-contracts/pull/16 @niveda-krish

Comment (niveda-krish) [reply src/cadence/scripts/getNFTs.cdc:406]: Thank you!

**Comment by prpatel05** [comment]
Hello!

With the release of Secure Cadence on Testnet (and soon to Mainnet on 2022-06-15), changes to your contracts are required to continue functioning.  You can find more information here on what is required - https://forum.onflow.org/t/breaking-changes-coming-with-secure-cadence-release/3052.  In the interim, NFT and NFT metadata retrieval will not function and will not be supported on the Alchemy API until these contracts are updated.

Thanks

**Pull request merged**